app/src/main/resources/logback-auditlog.xml

```diff
@@ -0,0 +1,46 @@
+<included>
+    <property name="AUDIT_LOG_DIR" value="/tmp/log/opintoni/auditlog"/>
```

Comment:
Audit-lokit olisi toistaiseksi hyvä lokittaa `/var/log/opintoni`-polun alle kuten muutkin lokit (ei välttämättä tarvita alikansiota, mutta eipä se haittaakaan)

Reply:
Mielestäni parempi olla omassa kansiossaan. -> Jätän

Reply:
Suurin pointti tuossa kommentissa oli tuo `/var/log/opintoni` eikä niinkään onko audit-lokit lokituskansion alla omassa alikansiossaan vaiko ei, millä ei ole juuri merkitystä. `/tmp`in alle audit-lokeja ei kuitenkaan haluta jättää.